set.mm

```diff
@@ -510554,6 +510862,1211 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+      ZVXQBEGVXJXWJIJWUKWUSXXAXRVXQVYKVYPFWVHXVIYAZVXQGFUVSVYKVXJVWJWUMXXBWURXT
+      WURVXQXWKVGVXQBEGVYKVXJIJWUIVXQFBVYKWUNXXBWHWUSYJYRYMVXQBCDEFGUVJVVOVXIVV
+      PVXJHIJKLMWUIWUMWVIWVDWVJWURWXQYNVXQBCUVIDEFGXUHWYGXUIVYEHIJKLMWUIWUMXVJW
+      YOXVKWVLUWLWRWTVXQVVJVVRVXKUVJWVMXEVXQXUEXUJWYDWYHUVIXVLWYQTWTVXQVUOVVJVW
+      OVXKUVJWVQWVRTVXQXUAXUEWXTWYDUVIXVMWYSTWTWWAXCWWBXCWWCXCAUVLUVGRUVMUVHRAU
+      VKUVKBFAFBUVKUWEUWTWHUWTVNUVGUVLDUXAVPVHUXIUXNUVMUXCUVJSZUXCVTUEUFBFUXSXX
+      CUVMUXMUVJSZUXCUXSUXCUXMUVMUVJUYAWEUXSUVKUXJGSZUYFPZDQUXMXXDUXCUXSXXFUXLD
+      UXSXXEUXJUYFUXKUXSBEGUVKUXJIJUWRUYHUYNWPUYRWQWSUXSBCDEFGUVJUVKUXJUVKUXKHI
+      JKLMUYTUYIUXSFBUVKUYJVUBWHZUYNVUBUYKWXQYNUYAWTWOVUCXCUXIUXNUXCUVMUVJSZUXC
+      VTUEUFBFUXSXXHUXMUXCUXSUXMUVMUVJSUYCVUHPZDQXXHUXMUXSBCDEFGUVJUXJUVKUXKUVK
+      HIJKLMUYTUYIUYNXXGUYKVUBWXQYNUXSUXCUXMUVMUVJUYAXEUXSUXLXXIDUXSUXJUYCUXKVU
+      HUXSUYCUXJUYOXQVUKWQWSWTUYAYTVUCXCAUXEVUNWXJVUMUXCUVJSZVTZVUQUXNXXKUEUFBF
+      VUTVVDXXKUGUHBFVVIWXLVVCUXMUVJSZWXJXXJVVIWXPVVAUXJGSZXWTPZDQWXLXXLVVIWXOX
+      XNDVVIWXNXXMVVPXWTVVIBEGUXJVVAIJVVTVWBVWCYJVVIBEGUXKVVBIJVVTVWIVWHYJWQWSW
+      XRVVIBCDEFGUVJVVAUXJVVBUXKHIJKLMVVTVWAVWCVWBVWEVWDWXQYNWTWXMVVIVUMVVCUXCU
+      XMUVJVVLVVKTWTVWMXCVWNXCYCUVF $.
+  $}
+
+  ${
+    rloc0g.1 $e |- .0. = ( 0g ` R ) $.
+    rloc0g.2 $e |- .1. = ( 1r ` R ) $.
+    rloc0g.3 $e |- L = ( R RLocal S ) $.
+    rloc0g.4 $e |- .~ = ( R ~RL S ) $.
+    rloc0g.5 $e |- ( ph -> R e. CRing ) $.
+    rloc0g.6 $e |- ( ph -> S e. ( SubMnd ` ( mulGrp ` R ) ) ) $.
+    ${
+      rloc0g.o $e |- O = [ <. .0. , .1. >. ] .~ $.
+      $( The zero of a ring localization.  (Contributed by Thierry Arnoux,
+         4-May-2025.) $)
+      rloc0g $p |- ( ph -> O = ( 0g ` L ) ) $=
+        ( cfv wcel co eqid syl cop cec c0g cgrp cbs cplusg wceq cmulr rloccring
+        crnggrpd cxp cqs grpidcl cmgp csubmnd ringidval subm0cl opelxpd ecelqsi
+        cerl ovexi csg wss mgpbas submss rlocbas rlocaddval crngringd ringridmd
+        ccrg eleqtrd oveq12d grplidd eqtrd sseldd ringlidmd opeq12d wa isgrpid2
+        eceq1d biimpa syl12anc eqtr4id ) AGHEUAZBUBZFUCPZOAFUDQZWEFUEPZQZWEWEFU
+        FPZRZWEUGZWFWEUGZAFACUEPZCUFPZBCDCUHPZFWNSZWPSZWOSZKLMNUIUJAWEWNDUKZBUL
+        ZWHAWDWTQWEXAQAHEWNDACUDQHWNQACMUJZWNCHWQIUMTZADCUNPZUOPQZEDQNDXDECEXDX
+        DSZJUPUQTZURWTWDBBCDUTLVAUSTAWNBCDWPFCVBPZVJWTHWQIWRXHSWTSKLMAXEDWNVCNW
+        NDXDWNCXDXFWQVDVETZVFVKAWKHEWPRZXJWORZEEWPRZUAZBUBWEAWNWOWJBCDWPHHEEFWQ
+        WRWSKLMNXCXCXGXGWJSZVGAXMWDBAXKHXLEAXKHHWORHAXJHXJHWOAWNCWPEHWQWRJACMVH
+        ZXCVIZXPVLAWNWOCHHWQWSIXBXCVMVNAWNCWPEEWQWRJXOADWNEXIXGVOVPVQVTVNWGWIWL
+        VRWMWHWJFWFWEWHSXNWFSVSWAWBWC $.
+    $}
+
+    ${
+      $d .1. a b x $.  $d .~ a b x $.  $d L a b x $.  $d R a b $.  $d S a b $.
+      $d a b ph x $.
+      rloc1r.i $e |- I = [ <. .1. , .1. >. ] .~ $.
+      $( The multiplicative unit of a ring localization.  (Contributed by
```

Comment:
"multiplicative identity" or "unity"

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/2b2a9e540a51ed9c7becc4b6bb3e36ae76cc7501